package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -220,28 +176,28 @@ wireless.foreach_radio(uci, function(radio, index, config)
 	uci:delete('wireless', radio_name, 'basic_rate')
 
 	local band = radio.band
+	-- now we assign radios to roles
+	local roles = uci:get('gluon', 'band_' .. band, 'role')
```

Comment:
```suggestion
	local roles = uci:get_list('gluon', 'band_' .. band, 'role')
```
This will set roles to `{}` if no roles are set, so you can omit the `if roles` checks later.